### PR TITLE
Regen

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
cc @EPashkin @sdroege

### CODE REVIEW

File: src/auto/application_command_line.rs [hunk 1]
```diff
@@ -145,7 +145,7 @@ impl<O: IsA<ApplicationCommandLine>> ApplicationCommandLineExt for O {
 
 unsafe extern "C" fn notify_is_remote_trampoline<P, F: Fn(&P) + 'static>(this: *mut ffi::GApplicationCommandLine, _param_spec: glib_ffi::gpointer, f: glib_ffi::gpointer)
 where P: IsA<ApplicationCommandLine> {
-    let f: &F = transmute(f);
+    let f: &F = &*(f as *const F);
```
Comment (EPashkin) [inline src/auto/application_command_line.rs]: @GuillaumeGomez this change also better applied to manual code, but I don't insist

Comment (EPashkin) [reply src/auto/application_command_line.rs]: with removing `#[cfg_attr(feature = "cargo-clippy", allow(transmute_ptr_to_ref))]` if there present

**Comment by EPashkin** [comment]
👍

File: src/auto/application_command_line.rs (hunk 1 above)
Comment (GuillaumeGomez) [reply src/auto/application_command_line.rs]: Updating.

Comment (GuillaumeGomez) [reply src/auto/application_command_line.rs]: This is the only actual case where the `transmute` can be easily changed without having a huge (and ugly) `as stuff`.

**Comment by sdroege** [comment]
:+1:

**Pull request merged**